modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java

```diff
@@ -166,6 +168,87 @@ public void testCreateBlog() throws Exception {
 		);
 	}
 
+	@Test
+	public void testCreateBlogPostWithImage() throws Exception {
+		String documentsHref = ApioClientBuilder.given(
```

Comment:
What do you think about move the creation of the documents to a Test Util? I Think Rubén is doing exactñy the same